app/src/main/java/pl/example/android/garbageapp/ui/BaseActivitySector.java

```diff
@@ -25,4 +35,25 @@ protected void onCreate(Bundle savedInstanceState) {
         DetailViewModelFactory factory = InjectorUtils.provideDetailViewModelFactory(currentSector().getApplicationContext(), sectorType());
         mViewModel = ViewModelProviders.of(currentSector(), factory).get(DetailActivityViewModel.class);
     }
+
+    protected boolean checkIfSectorMarkedAsNotification(){
+        int notificationSectorType = SectorType.toInt(SectorType.UNSET);
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(currentSector());
+        if(sharedPreferences.contains(NOTIFICATION_SECTOR_TYPE))
+            notificationSectorType = sharedPreferences.getInt(NOTIFICATION_SECTOR_TYPE, notificationSectorType);
+
+        return notificationSectorType == SectorType.toInt(sectorType());
+    }
+
+    protected void setNotification(View view){
```

Comment:
The same as above - method name markForNotification() for example is more readable for me.

Reply:
Good point. It was my bad day for finding names :)